### PR TITLE
created landing page

### PR DESCRIPTION
**Pull request by cnnebe**
Added some basic styling for the landing page, and the base page. There seems to be an issue with the index page, but I did not touch it. So, I am not sure what the problem is.

### CODE REVIEW

File: HandprintGenerator/static/handprintgenerator.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+}
+#custom-bootstrap-menu.navbar-default .navbar-toggle:hover .icon-bar,
+#custom-bootstrap-menu.navbar-default .navbar-toggle:focus .icon-bar {
+    background-color: #72c94a;
+}
+
+
+/* Styles go here */
+
+/* Sticky footer styles
```
Comment (jinjennifer) [inline HandprintGenerator/static/handprintgenerator.css]: Sticky footer is not working - this is why the index page looks weird.

File: HandprintGenerator/templates/base.html [hunk 2]
```diff
@@ -1,17 +1,59 @@
...
-        <link rel="stylesheet" href="{% static 'css/handprintgenerator.css' %}">
+        <link rel="stylesheet" href="{% static 'css/handprintgenerator.css' %}"> 
     </head>
     
     <body>
+        <div id="wrap">
+
+            <div id="custom-bootstrap-menu" class="navbar navbar-default " role="navigation">
+                <div class="container-fluid">
+                    <div class="navbar-header"><a class="navbar-brand" href="#">Handprinter</a>
```
Comment (jinjennifer) [inline HandprintGenerator/templates/base.html]: Needs to be linked to home page

File: HandprintGenerator/templates/base.html [hunk 3]
```diff
@@ -1,17 +1,59 @@
...
+                        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-menubuilder"><span class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></button>
+                    </div>
+                    <div class="collapse navbar-collapse navbar-menubuilder">
```
Comment (jinjennifer) [inline HandprintGenerator/templates/base.html]: I can't click on the collapsible nav bar to show the navigation elements.

File: HandprintGenerator/templates/HandprintGenerator/home.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  <div class="col-sm-6 col-md-4">
+    <div class="thumbnail">
+      <img src="..." alt="...">
+      <div class="caption">
+        <h3>What is Handprinter Image Space</h3>
+        <p><a href="#" class="btn btn-primary" role="button">What is Handprinter?</a> </p>
+      </div>
+    </div>
+  </div>
+</div>s
```
Comment (jinjennifer) [inline HandprintGenerator/templates/HandprintGenerator/home.html]: small typo, remove s

File: HandprintGenerator/templates/base.html [hunk 5]
```diff
@@ -1,17 +1,59 @@
...
 <html>
-    
+
     <head>
         <title>Handprinter</title>
         
         <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
         <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
-        <link rel="stylesheet" href="{% static 'css/handprintgenerator.css' %}">
+        <link rel="stylesheet" href="{% static 'css/handprintgenerator.css' %}">
```
Comment (jinjennifer) [inline HandprintGenerator/templates/base.html]: It doesn't look like the css is being called. I think it is missing type="text/css" but there still seem to be some issues.

File: HandprintGenerator/static/handprintgenerator.css [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+  min-height: 100%;
+  height: auto;
+  /* Negative indent footer by its height */
+  margin: 0 auto -60px;
+  /* Pad bottom by footer height */
+  padding: 0 0 60px;
+}
+
+/* Set the fixed height of the footer here */
+#footer {
```
Comment (jinjennifer) [inline HandprintGenerator/static/handprintgenerator.css]: This is not used anywhere

File: HandprintGenerator/templates/base.html [hunk 7]
```diff
@@ -1,17 +1,59 @@
...
+        </div>
+
+    <div class="panel panel-default">
+        <div class="panel-footer">
+            <nav>
+            <table>
+            <th><a href="/contact">Contact Us</a></th>
+            <th><a href="/privacy">Privacy Policy</a></th>
+            <th><a href="/terms">Terms of Service</a></th> 
+            <th><a href="/admin">Administration</a></th>
```
Comment (jinjennifer) [inline HandprintGenerator/templates/base.html]: Authentication to show this page should be put back in. Only show this to admins.

File: HandprintGenerator/templates/base.html [hunk 8]
```diff
@@ -1,17 +1,59 @@
...
+                    <div class="navbar-header"><a class="navbar-brand" href="#">Handprinter</a>
+                        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-menubuilder"><span class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span class="icon-bar"></span><span class="icon-bar"></span></button>
+                    </div>
+                    <div class="collapse navbar-collapse navbar-menubuilder">
+                        <ul class="nav navbar-nav navbar-left"><li><a href="/about">About Handprinter</a></li><li><a href="/index">Action Idea Index</a></li><li><a href="/new">New Action Idea Form</a></li><li><a href="/create_user">New User Form</a></li></ul><ul class="nav navbar-nav navbar-right"><li>{% if user.is_authenticated %}Welcome, {{ user.username }} <a href="/logout">Logout</a>{% else %}<a href="/login">Log In</a>{% endif %}</li></ul>
+                    </div>
+                </div>
+            </div>
+
+
```
Comment (jinjennifer) [inline HandprintGenerator/templates/base.html]: Add <section class="container"> here and remove from the templates

File: HandprintGenerator/templates/base.html (hunk 5 above)
Comment (mcvellaa) [inline HandprintGenerator/templates/base.html]: Yeah, she moved the CSS file. When I first set it up, it was working with the CSS file in the static/css folder. The static Django tag pulls up the static folder, and the relative address is saying that it's in the /css/ folder

File: HandprintGenerator/urls.py [hunk 9]
```diff
@@ -5,7 +5,7 @@
 
 app_name = 'HandprintGenerator'
 urlpatterns = [
-    url(r'^$', views.index, name='index'),
```
Comment (mcvellaa) [inline HandprintGenerator/urls.py]: Why is the index taken out??? We need to leave the action idea index......the home page is additional, not in place of the index

File: HandprintGenerator/templates/base.html (hunk 7 above)
Comment (mcvellaa) [inline HandprintGenerator/templates/base.html]: True.

File: HandprintGenerator/templates/base.html [hunk 10]
```diff
@@ -1,17 +1,59 @@
...
 <!DOCTYPE HTML>
 <html>
-    
+
     <head>
         <title>Handprinter</title>
         
         <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
         <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
-        <link rel="stylesheet" href="{% static 'css/handprintgenerator.css' %}">
```
Comment (mcvellaa) [inline HandprintGenerator/templates/base.html]: it looks like this is exactly the same line, even though it says that one was removed and another was added?

File: HandprintGenerator/urls.py (hunk 9 above)
Comment (jinjennifer) [inline HandprintGenerator/urls.py]: It wasn't taken out, the home page used to be the action idea index page, it's now an actual home page.

File: HandprintGenerator/templates/base.html (hunk 5 above)
Comment (jinjennifer) [inline HandprintGenerator/templates/base.html]: Shouldn't it be moved back into the css folder?

**Pull request merged**